ourjapanlife/findadoc-server
# 113 flatten migrations

Resolves #113


# What changed
Flatten all the migrations into one for clarity

# Testing instructions
`docker compose up`

And then in another terminal, run the app `yarn run dev`

Ensure it's happy
